pkg-example-smf-bp/3gpp.yaml

```diff
@@ -0,0 +1,59 @@
+kind: ConfigMap
+metadata:
+  name: smf-configmap
+  labels:
+    app: free5gc
+data:
+  smfcfg.yaml: |
+    info:
+      version: 1.0.2
+      description: SMF initial local configuration
+
+    configuration:
+      serviceNameList:
+        - nsmf-pdusession
+        - nsmf-event-exposure
+        - nsmf-oam
+
+      sbi:
+        scheme: http
+        registerIPv4: smf-nsmf # IP used to register to NRF
+        bindingIPv4: 0.0.0.0  # IP used to bind the service
+        port: 80
+        tls:
+          key: config/TLS/smf.key
+          pem: config/TLS/smf.pem
+
+      nrfUri: http://nrf-nnrf:8000
+      pfcp:
+        addr: 10.100.50.244
+      smfName: SMF
+      snssaiInfos:
+        - sNssai:
+            sst: 1
+            sd: 010203
+          dnnInfos: # DNN information list
+            - dnn: internet # Data Network Name
+              dns: # the IP address of DNS
+                ipv4: 8.8.8.8
```

Comment:
shouldn't this come from an IPAllocation or some other config? I guess that's getting into "workload configuration" at that point. We haven't figured out how to generate this file from KRM yet.

Reply:
DNN we have a reference for. the DNS name was not yet allocated, but we could create a reference to a DNS that was assigned. So far this is the config that SMF needs. So a number of parameters haven't been exposed. However DNS is easy.